src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,8 +1,8 @@
 databaseChangeLog:
   - include:
       file: /db/changelog/changes/01-create-table-books.yaml
-  - include:
-      file: db/changelog/changes/02-create-table-shoppingCart.yaml
+#  - include:
+#      file: db/changelog/changes/02-create-table-shoppingCart.yaml
```

Comment:
? shopping card should be created

Reply:
i created it in another file, but forgot change name